docs/reference/pyproject-settings.rst

```diff
@@ -63,6 +63,29 @@ use them and examples.
 
    Extra arguments to be passed to the ``meson install`` command.
 
+.. option:: tool.meson-python.wheel.exclude
+
+   List of glob patterns matching paths of files that must be excluded from
+   the Python wheel. The accepted glob patterns are the ones implemented by
+   the Python :mod:`fnmatch` with case sensitive matching. The paths to be
+   matched are as they appear in the Meson introspection data, namely they are
```

Comment:
It will be useful to name `intro-install_plan.json` explicitly, and perhaps link to https://mesonbuild.com/IDE-integration.html#install-plan. The average reader will be unaware of the existence of this file. Maybe also explain where it can be found, or use `<build-dir>/meson-info/intro-install_plan.json` as the name.